ntfy/cli.py

```diff
@@ -49,9 +49,10 @@ def send_notification(message, args, config):
         module = import_module('ntfy.backends.{}'.format(backend))
 
         try:
+            backend_config = config.get(backend, {})
+            backend_config.update(args.option)
```

Comment:
probably - unless all your command line options are OK getting passed to wherever as simple strings.  I needed to pass along a (string, int) tuple for server and an int for ssl_version, though I didn't even test this part of the library *at all* let alone passing other types - it might have handled it fine